includes/Admin/Settings_Screens/Advertise.php

```diff
@@ -0,0 +1,254 @@
+
+/**
+ * The Advertise settings screen object.
+ */
+class Advertise extends Admin\Abstract_Settings_Screen {
+
+
+	/** @var string screen ID */
+	const ID = 'advertise';
+
+
+	/**
+	 * Advertise settings constructor.
+	 *
+	 * @since 2.2.0-dev.1
+	 */
+	public function __construct() {
+
+		$this->id    = self::ID;
+		$this->label = __( 'Advertise', 'facebook-for-woocommerce' );
+		$this->title = __( 'Advertise', 'facebook-for-woocommerce' );
+
+		$this->add_hooks();
+	}
+
+
+	/**
+	 * Adds hooks.
+	 *
+	 * @since 2.2.0-dev.1
+	 */
+	private function add_hooks() {
+
+		add_action( 'admin_head', [ $this, 'output_scripts' ] );
+
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
+	}
+
+
+	/**
+	 * Enqueues assets for the current screen.
+	 *
+	 * @internal
+	 *
+	 * @since 2.2.0-dev.1
+	 */
+	public function enqueue_assets() {
+
+		if ( ! $this->is_current_screen_page() ) {
+			return;
+		}
+
+		wp_enqueue_style( 'wc-facebook-admin-advertise-settings', facebook_for_woocommerce()->get_plugin_url() . '/assets/css/admin/facebook-for-woocommerce-advertise.css', [], \WC_Facebookcommerce::VERSION );
+	}
+
+
+	/**
+	 * Outputs the LWI Ads script.
+	 *
+	 * @internal
+	 *
+	 * @since 2.1.0-dev.1
+	 */
+	public function output_scripts() {
+
+		$connection_handler = facebook_for_woocommerce()->get_connection_handler();
+
+		if ( ! $connection_handler || ! $connection_handler->is_connected() || ! $this->is_current_screen_page() ) {
+			return;
+		}
+
+		?>
+		<script>
+			window.fbAsyncInit = function() {
+				FB.init( {
+					appId            : '<?php echo esc_js( $connection_handler->get_client_id() ); ?>',
+					autoLogAppEvents : true,
+					xfbml            : true,
+					version          : 'v8.0',
+				} );
+			};
+		</script>
+		<?php
+	}
+
+
+	/**
+	 * Gets the LWI Ads configuration to output the FB iframes.
+	 *
+	 * @since 2.2.0-dev.1
+	 *
+	 * @return array
+	 */
+	private function get_lwi_ads_configuration_data() {
+
+		$connection_handler = facebook_for_woocommerce()->get_connection_handler();
+
+		if ( ! $connection_handler || ! $connection_handler->is_connected() ) {
+			return [];
+		}
+
+		return [
+			'business_config' => [
+				'business' => [
+					'name' => $connection_handler->get_business_name(),
+				],
+			],
+			'setup'           => [
+				'external_business_id' => $connection_handler->get_external_business_id(),
+				'timezone'             => $this->parse_timezone( wc_timezone_string(), wc_timezone_offset() ),
+				'currency'             => get_woocommerce_currency(),
+				'business_vertical'    => 'ECOMMERCE',
+			],
+			'repeat'          => false,
+		];
+	}
+
+
+	/*
+	 * Converts the given timezone string to a name if needed.
+	 *
+	 * @since 2.2.0-dev.1
+	 *
+	 * @param string $timezone_string Timezone string
+	 * @param int|float $timezone_offset Timezone offset
+	 * @return string timezone string
+	 */
+	private function parse_timezone( $timezone_string, $timezone_offset = 0 ) {
```

Comment:
No need to change it now, but I think the purpose of this method is the same as the `get_timezone_string()` helper in the Connection handler:

https://github.com/facebookincubator/facebook-for-woocommerce/blob/c1b264f0d685e930be91502d111acef374919f70/includes/Handlers/Connection.php#L846-L865

Reply:
@wvega yeah, no need to worry about that one on this PR though